MonoZelda/Enemies/EnemyClasses/Aquamentus.cs

```diff
@@ -64,6 +66,7 @@ public void EnemySpawn(SpriteDict enemyDict, Point spawnPosition, CollisionContr
             {
                 projectileDictionary.Add(projectile, new EnemyProjectileCollision(projectile, collisionController));
             }
+            EnemyHitbox.setEnemy(this);
```

Comment:
I know that for this sprint we're only worrying about implementing a few enemies for the sake of showcase, but I'm wondering if there's some way to generalize collision stuff that will need to be added to all enemy classes. Maybe a parent concrete or abstact class and call base methods in each enemy? We'll have to discuss this as a group.